application/templates/service.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.service }}
+{{- if or (.Values.service) (.Values.mutualTls.enabled) }}
```

Comment:
why is that so?

Reply:
we should not enable it when TLS is enabled. its like a requirement that in order to enable mTLS, service has to be enabled but this logic does not sound logical here